xml/System.Net.Sockets/Socket.xml

```diff
@@ -4219,10 +4219,11 @@ This method populates the <xref:System.Net.Sockets.Socket> instance with data ga
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets a <see cref="T:System.Boolean" /> value that specifies whether the <see cref="T:System.Net.Sockets.Socket" /> is a dual-mode socket used for both IPv4 and IPv6.</summary>
+        <summary>Gets or sets a value that specifies whether the <see cref="T:System.Net.Sockets.Socket" /> is a dual-mode socket used for both IPv4 and IPv6.</summary>
         <value>
-          <see langword="true" /> if the <see cref="T:System.Net.Sockets.Socket" /> is a  dual-mode socket; otherwise, <see langword="false" />. The default is <see langword="false" />.</value>
+          <see langword="true" /> if the <see cref="T:System.Net.Sockets.Socket" /> is a dual-mode socket; otherwise, <see langword="false" />. The default is <see langword="true" /> if the socket was created by calling the <see cref="M:System.Net.Sockets.Socket.#ctor(System.Net.Sockets.SocketType,System.Net.Sockets.ProtocolType)" /> constructor and the operating system supports dual-stack sockets; otherwise, the default is <see langword="false" />.</value>
```

Comment:
```suggestion
          <see langword="true" /> if the <see cref="T:System.Net.Sockets.Socket" /> is a dual-mode socket; otherwise, <see langword="false" />. The default is <see langword="true" /> if the socket was created by calling the <see cref="M:System.Net.Sockets.Socket.#ctor(System.Net.Sockets.SocketType,System.Net.Sockets.ProtocolType)" /> constructor and the operating system supports IPv6; otherwise, the default is <see langword="false" />.</value>
```